Makefile

```diff
@@ -127,17 +134,23 @@ lint: golangci-lint ## Run golangci-lint against code.
 
 ##@ Build
 
-.PHONY: manager
-manager: ## Build manager binary.
-	go build -o $@
+manager: $(shell find -name "*.go") go.mod go.sum  ## Build manager binary.
+	go build -o $@ ./cmd/manager
+
+manager-hub: $(shell find -name "*.go") go.mod go.sum  ## Build manager-hub binary.
+	go build -o $@ ./cmd/manager-hub
 
 .PHONY: run
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
 .PHONY: docker-build
-docker-build: unit-test ## Build docker image with the manager.
-	docker build -t $(IMG) .
+docker-build: ## Build docker image with the manager.
+	docker build -t $(IMG) --build-arg TARGET=manager .
+
+.PHONY: docker-build-hub
+docker-build-hub: ## Build docker image with the hub manager.
+	docker build -t $(IMG) --build-arg TARGET=manager-hub .
```

Comment:
I don't think `IMG` should be use here. Don't we want a different image name for the hub?

Reply:
Fixed, thanks!